build.gradle

```diff
@@ -301,7 +301,10 @@ allprojects {
     if (project.path.contains(":distribution:docker")) {
       enabled = false
     }
-
+    if (project.path.contains(":libs:cli")) {
```

Comment:
I assume we are just arbitrarily picking a project here, yes?

Reply:
yes